### PR TITLE
Added new svg logo for the app

### PR DESCRIPTION
**Pull request by imaryandokania**
## Purpose / Description
I have converted a png to svg.
images

This is added for future drawable additions.

## Approach
The svg is licensed under MIT license and Is compatible with GPL.

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Comment by Arthur-Milchior** [comment]
Hi

Thanks for the commit. I thought we already had a svg. I think @david-allison-1 followed this more closely than myself. In any case, thanks for converting it.

There are four commits in this PR. Not a huge trouble as I can squash. But it would be helpful for reviewer if you can limit the amount of commits, so that I don't spend time looking as useless data.

You may want in particular to get the habit of rebasing onto master instead of merging

File: docs/graphics/src/logo.svg [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<!--
+MIT License
+
+Copyright (c) [2021] [Anki Droid]
```
Comment (david-allison-1) [inline docs/graphics/src/logo.svg:4]: > The svg is licensed under MIT license and Is compatible with GPL.

Can you please confirm (ideally in public) that the author has explicitly agreed for the file to be licensed as such and provide copyright via either their name/web handle

**Review by david-allison-1 — CHANGES_REQUESTED** [review]
See comment - we need to be very careful about assigning licenses to files that we don't own.

I personally asked about using the icon a while back, so I'm expecting this'll be fine, but we need to be on the right side of the law

**Comment by imaryandokania** [comment]
> See comment - we need to be very careful about assigning licenses to files that we don't own.
> 
> I personally asked about using the icon a while back, so I'm expecting this'll be fine, but we need to be on the right side of the law

I have replied in discord as well that the creator has replied and was okay if we use the image , if you want I can share the conversation with the creator the logo .

File: docs/graphics/src/logo.svg (hunk 1 above)
Comment (imaryandokania) [reply docs/graphics/src/logo.svg:4]: Screenshot 2021-03-17 at 10 19 51 AM
He is the creator!

Comment (david-allison-1) [reply docs/graphics/src/logo.svg:4]: Sorry to be annoying, could you explicitly confirm that he's OK with licensing it under the MIT License for us.

**Comment by imaryandokania** [comment]
Yes I have messaged the creator regarding license as well .. waiting for
his reply

**Comment by Arthur-Milchior** [comment]
Ideally, if you can ask the author to confirm here, would allow to keep track and avoid to use you as a middle-man in case of need

**Comment by imaryandokania** [comment]
> Ideally, if you can ask the author to confirm here, would allow to keep track and avoid to use you as a middle-man in case of need

Okay I will tell him to comment it here

**Comment by imaryandokania** [comment]
I have rebased this, so won't be a problem now

**Comment by lucajulius** [comment]
Hi, I'm the original author of the logo and I'm happy to license it to AnkiDroid under the MIT license.

File: docs/graphics/src/logo.svg (hunk 1 above)
Comment (david-allison-1) [inline docs/graphics/src/logo.svg:4]:
```suggestion
Copyright (c) 2021 lucajulius
```

**Review by david-allison-1 — CHANGES_REQUESTED** [review]
Thanks!!!! Let's get this in

**Comment by imaryandokania** [comment]
> Thanks!!!! Let's get this in

Yeah !

**Review by david-allison-1 — APPROVED** [review]
LGTM. Requires squash merge

**Comment by imaryandokania** [comment]
> LGTM. Requires squash merge

do I rebase it again?

**Review by infinyte7 — APPROVED** [review]

**Comment by david-allison-1** [comment]
> do I rebase it again? So that you don't need to squash merge

It'd be appreciated. You can perform a squash as part of a rebase.

Do try and inform maintainers if you're making major changes after an "accepted" review

**Comment by imaryandokania** [comment]
@david-allison-1 It is rebased.Please review again

**Comment by imaryandokania** [comment]
> > do I rebase it again? So that you don't need to squash merge
> 
> It'd be appreciated. You can perform a squash as part of a rebase.
> 
> Do try and inform maintainers if you're making major changes after an "accepted" review

I have rebased and also added comment for review

**Review by david-allison-1 — APPROVED** [review]
Still looks good!

**Review by mikehardy — APPROVED** [review]
did not realize this was waiting on a second approval sorry :-)
Thank you!

**Comment by nwwt** [comment]
Two questions to add, hope it's ok to add them after already merging…

* The file is huge for a svg + rather simple geometric shapes (almost 10x anki.svg, despite that being more complex)? Probably because the lines are wobbly.
* Shouldn't the outside of a circle be transparent? It'll stick out on non-white backgrounds.

**Comment by Arthur-Milchior** [comment]
@nwwt 
The first point do not seems to be a question. If you can make a SVG as nice and with lower size, that'd be great. 
If you know how to replace while by transparent background it would be nice. I clearly missed that, thanks fo pointi this out.

**Pull request merged**